### ISSUE TITLE
Incorrect latest version detection for Packagist project

### ISSUE THREAD

**Issue by legoktm** [fedora-infra/anitya#1084]
See https://release-monitoring.org/project/192663/ - it thinks 0.3.2 is the latest version, not 0.5.

It's not explicitly stated on https://getcomposer.org/doc/articles/versions.md but versions like `0.5` are normalized to `0.5.0` for semver comparison purposes.

**Comment by legoktm** [comment]
I probably should have searched a bit harder to find the already-existing https://release-monitoring.org/project/11166/ but I note that one has an issue too, it doesn't notice the latest 0.5 release.

**Comment by Zlopez** [comment]
I'm using the [semver](https://pypi.org/project/semver/) python library for semantic versioning, so if you think this implementation is not right, report a bug against it.

I switched the versioning to RPM, which doesn't have any issue with this version scheme and treats `0.5` as latest. I also looked at the second project and this looks like it's pointing to some fork on GitHub which really doesn't have `0.5` version, so the setup is correct.
